open-broker-common/src/main/java/de/fraunhofer/iais/eis/ids/broker/core/common/persistence/RegistrationHandler.java

```diff
@@ -113,24 +120,31 @@ else if (msg instanceof ConnectorUnavailableMessage) {
             if(e.getMessage() == null)
             {
                 try {
+                	
                     e = new Exception(e.getCause().getMessage());
+                    logger.error("Failed to process message.", e);
                 }
                 catch (NullPointerException ignored)
                 {
                     e = new Exception(e.getClass().getName() + " with empty message.");
+                    logger.error(e.getClass().getName() + " with empty message.");
                 }
             }
             //Some unknown error has occurred, returning an internal error
+            logger.error("Internal Recipient Error ");
```

Comment:
Please add the exception to this log. Internal recipient errors need to be debugged, and a stack trace in the logs would greatly help here